src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -577,16 +570,16 @@ static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, float vx, float
                     if ((*child)->type == SvgNodeType::Use)
                         scene->push(_useBuildHelper(*child, vx, vy, vw, vh, svgPath));
                     else
-                        scene->push(_sceneBuildHelper(*child, vx, vy, vw, vh, svgPath));
+                        scene->push(_sceneBuildHelper(*child, vx, vy, vw, vh, svgPath, false));
                 } else if ((*child)->type == SvgNodeType::Image) {
                     auto image = _imageBuildHelper(*child, vx, vy, vw, vh, svgPath);
                     if (image) scene->push(move(image));
-                } else {
-                    auto shape = _shapeBuildHelper(*child, vx, vy, vw, vh);
+                } else if ((*child)->type != SvgNodeType::Mask) {
```

Comment:
I don't think this change is necessary if MaskType is added to isGroup.

Reply:
If Mask is added to the isGroup, I'll jump into the l.576 (569) and it's not what I want. 
I added this condition as the _shapeBuildHelper won't be called, but it's not necessary since the result of _shapeBuildHelper for the mask will be false - it's just for the performance